apps/editing-toolkit/editing-toolkit-plugin/block-patterns/src/premium-block-patterns.tsx

```diff
@@ -1,10 +1,11 @@
 /**
  * External dependencies
  */
-import * as React from 'react';
+import React from 'react';
```

Comment:
`import *` is not needed anymore with `esModuleInterop` enabled (https://www.typescriptlang.org/tsconfig#esModuleInterop)

Reply:
Should we change all instances of `import * as React from 'react';` to  `import React from 'react';` in all files?

Reply:
They should be changed in all ETK. Other apps still use them, I haven't validated their `tsconfig.json` yet. Probably for a future PR.

Reply:
Got it — thank you for the explanation!